github_info_swiss.py

```diff
@@ -0,0 +1,153 @@
+        )
+    except psycopg2.Error as e:
+        print(f"Connecting to Postgres: an error occurred while trying to connect to the database: {e}")
+        return None
+
+
+def extract_pull_links(cur):
+    print("Extracting links...")
+    try:
+        cur.execute('SELECT "Auto PR URL" FROM orphaned_prs_swiss;')
+        pull_links = [row[0] for row in cur.fetchall()]
+        return pull_links
+    except Exception as e:
+        print(f"Extracting pull links: an error occurred while extracting pull links from the database: {str(e)}")
+
+
+def get_auto_prs(repo_name, access_token, pull_links):
+    headers = {"Authorization": f"Bearer {access_token}"}
+    url = f"https://api.github.com/repos/opentelekomcloud-docs/{repo_name}/pulls"
+    params = {"state": "all"}
+    try:
+        response = requests.get(url, headers=headers, params=params)
+        response.raise_for_status()
+    except requests.exceptions.RequestException as e:
+        print(f"Get PRs: an error occurred while trying to get pull requests: {e}")
+    auto_prs = []
+    for pr in response.json():
+        body = pr.get("body")
+        if body and any(link in body for link in pull_links):
+            auto_prs.append(pr)
+    return auto_prs
+
+
+def add_github_columns(cur, conn):
+    print("Add info to the Postgres...")
+    try:
+        cur.execute(
+            '''
+            ALTER TABLE orphaned_prs_swiss
+            ADD COLUMN IF NOT EXISTS "Github PR State" VARCHAR(255),
+            ADD COLUMN IF NOT EXISTS "Github PR Merged" BOOLEAN;
+            '''
+        )
+        conn.commit()
+    except requests.exceptions.RequestException as e:
+        print(f"Add new column: an error occurred while trying to get pull requests: {e}")
+
+
+def update_orphaned_prs(cur, conn, rows, auto_prs):
+    print("Processing orphaned PRs...")
+    for row in rows:
+        pr_id, pull_link = row
+        gitea_repo_name = re.search(r"/docs-swiss/(.+?)/", pull_link).group(1)
+
+        matching_pr = None
+        for pr in auto_prs:
+            github_repo_name = pr["base"]["repo"]["name"]
+            if gitea_repo_name == github_repo_name:
+                matching_pr = pr
+                break
+        if matching_pr:
+            state = matching_pr["state"]
+            if matching_pr["merged_at"] is None:
+                merged = False
+            else:
+                merged = True
+            try:
+                cur.execute(
+                    'UPDATE orphaned_prs SET "Github PR State" = %s, "Github PR Merged" = %s WHERE id = %s;',
+                    (state, merged, pr_id)
+                )
+            except Exception as e:
+                print(f"Orphanes: an error occurred while updating orphaned PRs in the database: {str(e)}")
+
+        else:
+            continue
+
+    conn.commit()
+
+
+def main():
+    check_env_variables()
+    g = Github(github_token)
+
+    org = g.get_organization("opentelekomcloud-docs-swiss")

```

Comment:
this argument should be passed through environment variables to the original script to avoid double code